package.json

```diff
@@ -17,7 +17,7 @@
     "@exercism/active-background": "^0.6.2",
     "@exercism/codemirror-lang-wren": "https://github.com/exercism/codemirror-lang-wren",
     "@exercism/twine2-story-format": "https://github.com/exercism/twine2-story-format.git",
-    "@hotwired/turbo-rails": "^7.0.0-rc.1",
+    "@hotwired/turbo-rails": "^7.1.3",
```

Comment:
This fixed an issue with `Turbo.clearCache()` not working, but in the end I didn't use that. I've still kept this in, as the previous version was a release candidate.